packages/vertexai/src/api.ts

```diff
@@ -21,14 +21,20 @@ import { getModularInstance } from '@firebase/util';
 import { DEFAULT_LOCATION, VERTEX_TYPE } from './constants';
 import { VertexAIService } from './service';
 import { VertexAI, VertexAIOptions } from './public-types';
-import { ERROR_FACTORY, VertexError } from './errors';
-import { ModelParams, RequestOptions } from './types';
+import {
+  ModelParams,
+  RequestOptions,
+  VertexAIErrorCode
+} from './types';
+import { VertexAIError } from './errors';
 import { GenerativeModel } from './models/generative-model';
 
 export { ChatSession } from './methods/chat-session';
 
 export { GenerativeModel };
 
+export { VertexAIError } from './errors';
```

Comment:
Nit: You don't need the `from './errors'` here because you imported it above.

Reply:
Done